nemo/collections/multimodal/models/multimodal_llm/neva/neva_model.py

```diff
@@ -467,8 +471,10 @@ def forward(
         media = kwargs.pop('media', None)
         if parallel_state.is_pipeline_first_stage(ignore_virtual=True):
             self.embedding.word_embeddings.set_media(media)
+        torch.cuda.nvtx.range_push("LLM Model")
         result = GPTModel.forward(self, *args, **kwargs)
         torch.cuda.nvtx.range_pop()
+        torch.cuda.nvtx.range_pop()
```

Comment:
this is popped twice, now

Reply:
One pop is for the `LLM Model` and one pop is for the `neva_fwd`

Reply:
ooh, oops, the diff rendered weird and it broke my brain. sorry about that!